src/components/BackButton/BackButton.tsx

```diff
@@ -0,0 +1,26 @@
+import { StyledBackButton } from './StyledBackButton';
+import { ButtonProps } from '../Button/Button';
```

Comment:
Extending the `ButtonProps` means that there will also be an optional prop called `"variant"` on your BackButton component. This prop isn't really necessary for the functionality or styling of the BackButton, so let's just use `ButtonHTMLAttributes<HTMLButtonElement>` as the base interface.

```suggestion
```